doc/user_guide/user_guide.md

```diff
@@ -109,13 +109,30 @@ Transformers Extension Package. See [the latest release](https://github.com/exas
       --language-alias <LANGUAGE_ALIAS> \ 
       --version <RELEASE_VERSION> \
       --ssl-cert-path <ssl-cert-path> \
-      --use-ssl-cert-validation
+      --use-ssl-cert-validation \
   ```
 The `--ssl-cert-path` is optional if your certificate is not in the OS truststore. 
 The option `--use-ssl-cert-validation`is the default, you can disable it with `--no-use-ssl-cert-validation`.
 Use caution if you want to turn certificate validation off as it potentially lowers the security of your 
 Database connection.
 
+By default, the above command will upload and activate the language container at the System level.
+The latter requires you to have the System Privileges, as it will attempt to change DB system settings.
+If such privileges cannot be granted the activation can be skipped by using the `--no-alter-system` option.
+The command will then print the language activation SQL query, which looks like this:
+```sql
+ALTER SESSION SET SCRIPT_LANGUAGES=...
```

Comment:
Should we also print the alter system, with the message you can give this to your admin

Reply:
Not sure. It can be a bit confusing. From our standpoint, everything looks very straightforward. But a customer may get disorientated. If you don't fully understand what's going on it's always easier to follow steps without branching out.

Reply:
Just a thought. I don't have a strong opinion on that.

Reply:
in exaslct we are using the following text, what do you think?

https://github.com/exasol/script-languages-container-tool/blob/a45963b14bc228de0bd89d6b9c32dd4433173cd9/exasol_script_languages_container_tool/lib/tasks/upload/upload_container_base_task.py#L60C1-L72C17